web/profiles/joinup/config/install/file.settings.yml

```diff
@@ -0,0 +1,6 @@
+description:
+  type: 'textfield'
+  length: 128
+icon:
+  directory: 'core/modules/file/icons'
+make_unused_managed_files_temporary: true
```

Comment:
Yeah let's make sure old stuff gets cleaned up, or we will end up with jiggabytes of cruft.